src/main/scala/org/broadinstitute/dsde/rawls/webservice/AdminApiService.scala

```diff
@@ -84,6 +84,13 @@ trait AdminApiService extends HttpService with PerRequestCreator with UserInfoDi
       }
     } ~
     path("admin" / "groups" / Segment / "members") { (groupName) => //add members to group
+      put {
+        entity(as[RawlsGroupMemberList]) { memberList =>
+          requestContext => perRequest(requestContext,
+            UserService.props(userServiceConstructor, userInfo),
+            UserService.OverwriteGroupMembers(RawlsGroupRef(RawlsGroupName(groupName)), memberList))
```

Comment:
We need an endpoint that only adds and does not remove.  Our existing POST does that, so I would prefer to keep it as-is.